documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -52,7 +52,7 @@ However, you can configure the `deployment-scanner` module to deploy web applica
 [[monitored-directories]]
 == Monitored Directories
 
-The `deployment-scanner` module scans the deploy directory specified by the `jetty.deploy.monitoredDir` property, by default `$JETTY_BASE/webapps`, for web applications to deploy.
+The `deployment-scanner` module scans the web applications deploy directory specified by the `jetty.deploy.webappsDir` property, by default `$JETTY_BASE/webapps`, for web applications to deploy.
```

Comment:
```suggestion
The `deployment-scanner` module scans the web applications deploy directory specified by the `jetty.deploy.webappsDir` property, by default `$JETTY_BASE/webapps`, for web applications and core applications to deploy.
```

OR

```suggestion
The `deployment-scanner` module scans the web applications deploy directory specified by the `jetty.deploy.webappsDir` property, by default `$JETTY_BASE/webapps`, for contexts to deploy.
```

You might want to establish this language somewhere that a deployable context is a ContextHandler that could be a Webapp, coreapp, staticapp or directly configured core ContextHandler.